core/src/main/java/com/amplifyframework/predictions/models/Celebrity.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Holds the celebrity detection results
+ * for the predictions category.
+ */
+public final class Celebrity extends Feature<String> {
+
+    private Celebrity(final Builder builder) {
+        super(builder);
+    }
+
+    @Override
+    @NonNull
+    public String getTypeAlias() {
```

Comment:
Wait I forget, why not just return `FeatureType` form this interface? The `String` discards your constraint set.

Reply:
The point of this method is that some primitive feature detection like [`BinaryFeature`](https://github.com/aws-amplify/amplify-android/blob/identify-aws/core/src/main/java/com/amplifyframework/predictions/models/BinaryFeature.java) can specifically label what type of feature it is detecting.